jetty-ee8/jetty-ee8-nested/pom.xml

```
@@ -9,7 +9,7 @@
   </parent>
   <artifactId>jetty-ee8-nested</artifactId>
   <name>EE8 :: Nested</name>
-  <description>The jetty core handler adapted to legacy ee8 handler.</description>
+  <description>Jetty core handler adapter for EE8 (javax.servlet) environments.</description>
```

This is not core, nor is it handler specific.

```suggestion
  <description>Jetty EE8 (javax.servlet) nested implementation.</description>
```